tests/system/test_cluster/test_agent_files_deletion/test_agent_files_deletion.py

```diff
@@ -82,12 +108,14 @@ def test_agent_files_deletion(register_agent):
             assert not result, f'This file should not exist in {host} but it was found: ' \
                                f'{file["path"].format(id=agent_id, name=agent_name)}'
 
-    # Check that agent information does not exist anymore in global.db
+    # Check that agent information is not in the wdb socket
     for host in managers_hosts:
-        for query in db_queries:
-            result = host_manager.run_command(
-                host,
-                f'sqlite3 {join(WAZUH_PATH, "queue", "db", "global.db")} "{query.format(id=agent_id, name=agent_name)}"'
-            )
-            assert not result, f'This db query should have not returned anything in {host}, but it did: ' \
-                               f'{query.format(id=agent_id, name=agent_name)} -> {result}'
+        for query in queries:
+            result = host_manager.run_command(host,
+                                              f"{WAZUH_PATH}/framework/python/bin/python3.9 "
+                                              f"{script_path}/get_wdb_agent.py "
+                                              f"'{query.format(id=agent_id)}'")
+        assert not result, f'This db query should have not returned anything in {host}, but it did: {result}'
```

Comment:
```suggestion
        for query in queries:
            result = host_manager.run_command(host,
                                              f"{WAZUH_PATH}/framework/python/bin/python3.9 "
                                              f"{data_path}/get_wdb_agent.py "
                                              f"'{query.format(id=agent_id)}'")
        assert not result, f"This db query should have not returned anything in {host}, but it did: {result}"
```

Again, we are just checking the result of the last query